templates/wiki/wiki_footer.html

```diff
@@ -0,0 +1,37 @@
+{% load static %}
+{% include 'common/common_fonts.html' %}
+
+<div class="footer bg-light text-muted small pt-4">
+    <div class="container">
+    <div class="row pb-2">
+            <div class="col-lg-2">
+                <ul class="list-unstyled">
+                    <li><a href="">Home</a></li>
+                    <li><a href="">Public apps</a></li>
+                    <li><a href="">Public models</a></li>
+                    <li><a href="">User guide</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">User support</a></li>
+                    <li><a href="">Privacy policy</a></li>
```

Comment:
Only because they are also empty in the main footer template. I think some of these pages are not yet created. We will need to edit these links as part of the larger UI rework.

Reply:
Got it!

May be we could leave a `TODO:` here as well?